### PR TITLE
MSD example where multiple replicates are combined

### PR DESCRIPTION
**Pull request by markvrma**
Solves #3578 
Editing CHANGELOG is causing issues with mergibility ( #3630 )
Changes made in this Pull Request:
 - Added an example where multiple replicates are combined to docs ( MDAnalysis/analysis/msd.py )
 - Made requested changes in #3630 

PR Checklist
------------
 - [x] Docs
      - Added an example where multiple replicates are combined in docs. 
 - [x] Issue raised/referenced
      - #3578

### CODE REVIEW

**Comment by hmacdope** [comment]
I'm copying and pasting my comment from your closed PR:

@markvrma, please stick to a single PR per issue and don't open and close a bunch of them.
There is a few reasons for this:

* It is confusing for us as to what is active
* Comments are lost, meaning we have to re-review
* It makes it hard to keep track of who is working on what

All good for now, you can open a new PR for this issue, just so you know going forward.

If the reason for deleting them is git issues, don't worry, we squash and merge most PRs meaning the ugly history can be squashed into one commit. :)

**Comment by markvrma** [comment]
@hmacdope Apologies for the issue caused, moving forward I'll be sure to follow the above mentioned.

**Review by hmacdope — CHANGES_REQUESTED** [review]
Looks good, can you please add yourself to AUTHORS and add a CHANGELOG entry.

**Comment by markvrma** [comment]
Done!

File: package/CHANGELOG [hunk 1]
```diff
@@ -49,6 +49,8 @@ Enhancements
     monatomic ion charges or edge cases with nitrogen, sulfur, phosphorus and
     conjugated systems should now have correctly assigned bond orders and
     charges.
+  * Added MSD example where multiple replicates are combined(Issue #3578
```
Comment (hmacdope) [inline package/CHANGELOG]: Fix the indentation here please.

File: package/MDAnalysis/analysis/msd.py [hunk 2]
```diff
@@ -181,6 +181,29 @@
...
 We have now computed a self-diffusivity!
 
+Combining Multiple Replicates
+--------------------------------
+It is common practice to combine replicates when calculating MSDs. An example
+of this is shown below using MSD1 and MSD2.
+
+.. code-block:: python
+
+    u1 = mda.Universe(RANDOM_WALK, RANDOM_WALK_TOPO)
```
Comment (hmacdope) [inline package/MDAnalysis/analysis/msd.py]: Arguments to universe are wrong way round, this was my bad from original docs.

File: package/MDAnalysis/analysis/msd.py [hunk 3]
```diff
@@ -181,6 +181,29 @@
...
+    MSD1 = msd.EinsteinMSD(u1, select='all', msd_type='xyz', fft=True)
+    MSD1.run()
+
+    u2 = mda.Universe(RANDOM_WALK, RANDOM_WALK_TOPO)
```
Comment (hmacdope) [inline package/MDAnalysis/analysis/msd.py]: Same here for arguments wrong way round.

**Review by hmacdope — CHANGES_REQUESTED** [review]

File: package/CHANGELOG (hunk 1 above)
Comment (IAlibay) [reply package/CHANGELOG]: Also - new entries go first in the list not last.

File: package/MDAnalysis/analysis/msd.py [hunk 4]
```diff
@@ -181,6 +181,29 @@
...
+    MSD2 = msd.EinsteinMSD(u2, select='all', msd_type='xyz', fft=True)
+    MSD2.run()
+
+    combined_msds = np.concatenate((MSD1.results.msds_by_particle,
```
Comment (hmacdope) [inline package/MDAnalysis/analysis/msd.py:199]: Take back what I said earlier, you are concatenating and also averaging on the wrong axis.

**Review by hmacdope — CHANGES_REQUESTED** [review]

File: package/MDAnalysis/analysis/msd.py (hunk 2 above)
Comment (markvrma) [reply package/MDAnalysis/analysis/msd.py]: Should I fix the arguments to universe in `Computing an MSD`?

Comment (hmacdope) [reply package/MDAnalysis/analysis/msd.py]: Yeah that would be great :)

File: package/CHANGELOG (hunk 1 above)
Comment (markvrma) [reply package/CHANGELOG]: My bad, I've fixed it now. Thanks for the heads up!

File: package/AUTHORS [hunk 5]
```diff
@@ -190,7 +190,7 @@ Chronological list of authors
   - Aditi Tripathi
   - Sukeerti T
   - Robot Jelly
-
```
Comment (hmacdope) [inline package/AUTHORS:193]: Don't remove this blank line please. Once that is done all good to go.

**Review by hmacdope — CHANGES_REQUESTED** [review]

File: package/AUTHORS (hunk 5 above)
Comment (markvrma) [reply package/AUTHORS:193]: Yep I've added it back.

**Review by hmacdope — APPROVED** [review]
Approved pending CI! Congrats @markvrma :)

**Pull request merged**